js_modules/dagster-ui/packages/ui-core/src/dg-docs/__stories__/DGDocs.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import {Meta} from '@storybook/react';
+import {DGDocs} from '../DGDocs';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'dg docs/Page',
+  component: DGDocs,
+} as Meta;
+
+export const Default = () => {
+  return <DGDocs />;
+};
```

Comment:
Super basic Storybook example to demonstrate the `@dagster-io/dg-docs-components` import.